ci-operator/step-registry/openshift/e2e/openstack/csi/manila/openshift-e2e-openstack-csi-manila-workflow.yaml

```diff
@@ -14,6 +14,12 @@ workflow:
       CONFIG_TYPE: "minimal"
       BASE_DOMAIN: shiftstack.devcluster.openshift.com
       TEST_CSI_DRIVER_MANIFEST: manifest-openstack-manila.yaml
+      # We have negative tests which provoke errors and make sure
+      # we have them, so we don't want to run this test which
+      # always fails since we have these errors on purpose.
+      # https://bugzilla.redhat.com/show_bug.cgi?id=1990190
+      TEST_SKIPS: >-
+        events should not repeat pathologically
```

Comment:
Did you try to skip the offending test instead? Or would it still produce the events?

Reply:
I'm still investigating this issue, I skipped the test locally and I still see this "events should not repeat" test failing...

Reply:
BTW we can't skip this test, it seems. 

```
Error: no tests were found matching the TEST_SKIPS regex:
events should not repeat pathologically
```